Samples/InpaintHTTP/TestWebsite/index.html

```diff
@@ -2,15 +2,177 @@
+		}
+		@keyframes scanning {
+		  from {filter: blur(8px);}
+		  to {filter: blur(14px);}
+		}
+		@keyframes mask-fade {
+		  from {opacity: 1;}
+		  to {opacity: 0;}
+		}
+		.mask-finished
+		{
+			animation-name: mask-fade;
+			animation-duration: 4s;
+			animation-iteration-count: 1;
+			animation-timing-function: ease-in-out;
+			animation-fill-mode: forwards;
+		}
+		.btn-hide
+		{
+			visibility: hidden;
+		}
+	</style>
 </head>
 <body>
-	<center>
-		<h1>Local Inpainting Example</h1>
-		<form enctype="multipart/form-data" action="http://localhost:8069/api/inpaint" method="POST">
+	<div id="content">
+		<h1>Inpainting Example</h1>
+		<form enctype="multipart/form-data" method="POST">
 			<input type="hidden" name="MAX_FILE_SIZE" value="100000" />
-				Image: <input name="Image" type="file" /><br />
-				Mask: <input name="Mask" type="file" /><br />
-			<input type="submit" value="Upload File" />
+			<tt>Image: <input name="Image" id="Image" type="file" onchange="UpdateImg(this, false);"/><br>
+			Mask:&nbsp; </tt><input name="Mask" id="Mask" type="file" onchange="UpdateImg(this, true);"/></tt>
+			
+			<br><br>
+			
+			<div id="preview">
+				<img id="mask" class="mask-finished" alt="">
+			</div>
+			
+			<div id="footer"><a href="https://github.com/zavolokas/Inpainting">GitHub</a></div>
+			
+			<input type="button" id="inpaint" value="inpaint" />
 		</form>
-	</center>
-</body>
\ No newline at end of file
+	</div>
+	<script>
+function ProcessImage(){
+	$.post("/api/inpaint", function( data ) {
+		('#preview').css("background-image", "url('" + data + "')");
+	});
+}
+function UpdateImg(input, isMask) {
+	if (input.files && input.files[0]) {
+		var reader = new FileReader();
+			reader.onload = function (e) {
+				if(isMask)
+				{
+					$('#mask')
+						//.width(500)
+						//.height(650)
+						.removeClass("mask-finished")
+						.attr('src', e.target.result);
+				}
+				else
+				{
+					$('#preview')
+						.css("background-image", "url('" + e.target.result + "')");
+				}
+			};
+		reader.readAsDataURL(input.files[0]);
+	}
+}
+
+$(function(){
+
+    $("#inpaint").click(function(){
+
+        var fd = new FormData();
+
+        var imagefile = $('#Image')[0].files[0];
+		var maskfile = $('#Mask')[0].files[0];
+	
+        fd.append('file',imagefile);
+        fd.append('file',maskfile);
+		
+		console.log("Started Request");
+		$('#preview').addClass("img-busy");
+		$("#inpaint").addClass("btn-hide");
+		
+        $.ajax({
+            url: '/api/inpaint', //NOTE: you will get a JS header error when the url is NOT from the same hoste!
+            type: 'post',
+            data: fd,
+            contentType: false,
+            processData: false,
+            success: function(data){
+				console.log("Done");
+				console.log(data);
+				$("#inpaint").removeClass("btn-hide");
+				$("#mask")
+					.addClass("mask-finished");
+				$('#preview')
+					.css("background-image", "url('" + data + "')")
```

Comment:
I guess the data should be a raw image data, so it should start with `data:image/png;base64,` and then you append it with the image data in base64 format.

```suggestion
					.css("background-image", "url('data:image/png;base64," + data + "')")
```

Now that you have it here in base64, the only thing you need to do is to change the REST API part so that it actually returns base64 encoded bytes and you are good! 👍